api/features/serializers.py

```diff
@@ -98,6 +98,17 @@ class FeatureQuerySerializer(serializers.Serializer):
         required=False,
         help_text="Integer ID of the environment to view features in the context of.",
     )
+    state_enabled = serializers.BooleanField(
+        allow_null=True,
+        required=False,
+        default=None,
+        help_text="Boolean value to filter features as enabled or disabled.",
+    )
+    state_search = serializers.CharField(
```

Comment:
I'm being picky here but I think I'd rather this was `value_search`

Reply:
Ok done and as discussed, we changed `state_enabled` to `is_enabled` at the same time.